inspire_utils/name.py

```diff
@@ -105,8 +103,9 @@ def first(self):
 
     @property
     def first_initials_list(self):
-        names = self.first_list
-        return [(name[0] + u'.') for name in names if name]
+        names_no_dash_list = [name.split("-") for name in self.first_list]
+        names_no_dash_unpacked = [part_name for name in names_no_dash_list for part_name in name]
```

Comment:
you can use `itertools.chain.from_iterable for this:
```suggestion
        names_no_dash_unpacked = itertools.chain.from_iterable(name.split("-") for name in self.first_list)
```